File: C++/Coin Flipper.cpp
#include <iostream>
#include <cstdlib>
using namespace std;
int main()
{
	double h,f,t,s=0,r,x=0,y;
	cout <<"How many heads / flips / trials? (Separate with spaces.)\n";
	cin >>h>>f>>t;
	for(x;x<t;x++)
	{
		r = 0;
		for(y=0;y<f;y++)
			r += rand()%2?1:0;
		s += r==h?1:0;
	}
	cout <<"This combination will happen "<<s/t*100<<"% of the time.";
}

File: Python/No GUI/CoinFlipper.py
from random import random
times = int(input("How many tests?\n"))
flips = int(input("How many heads?\n"))
flips2 = int(input("How many flips?\n"))
times2 = 0
for x in range(times):
    r = 0
    for x in (random() for x in range(flips2)):
        r += 1 if x >=.5 else 0
    times2 += 1 if r == flips else 0
print("This combination will happen {0}% of the time.".format(times2/times*100))
